torch/nn/modules/rnn.py

```diff
@@ -507,6 +507,23 @@ class LSTM(RNNBase):
         All the weights and biases are initialized from :math:`\mathcal{U}(-\sqrt{k}, \sqrt{k})`
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
+    .. warning::
+        There are known deterministic issues for LSTM using cuDNN 7.6.5, 8.0 on CUDA 10.1 or later.
+        If you encounter this and need deterministic behavior,
+        a temporary workaround is to set environment variables.
```

Comment:
How "temporary" is it, actually? Is there a fix coming?

Reply:
This sentence should make it clear you're going to introduce the environment variables: "You can enforce deterministic behavior by setting the following environment variables:", for example.